### ISSUE TITLE
Publish API Docker image

### ISSUE THREAD

**Issue by pfwd** [howToCodeWell/code-quiz#103]

**Comment by harsh18262** [comment]
hi,do you need any help in creation of docker image?

**Comment by pfwd** [comment]
@harsh18262 yes please. I still need to refine this ticket. I will let you know when it’s ready

**Comment by pfwd** [comment]
> hi,do you need any help in creation of docker image?

Hi @harsh18262 I've created the ticket #150 which is ready for a PR
